gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -332,6 +336,9 @@ int main(int argc, char* argv[]) {
       }
     }
 
+    if (report_progress)
```

Comment:
I think reporting each 100th file would be sufficient and it would create less noise in the output. Of course, depending on the speed - but if we process 1000s of blocks per second, this might be overwhelming.

Perhaps, we could even turn the current flag into a flag saying after how many blocks to report progress (and turn it off by setting it to a high value, e.g. maxint).

Reply:
Good point. Updated this to rename the flag `report_progress_every` that takes an unsigned integer value and reports status after every n blocks.

The exegesis annotator definitely doesn't run that fast (maybe a couple blocks a second from my testing), so it's not an issue there, but using Owen's annotator implementation would definitely end up dumping a massive amount of text very quickly to the console.